test/root.test.ts

```diff
@@ -6,7 +6,8 @@ g.columns = 80
 import Help from '../src'
 
 const VERSION = require('../package.json').version
-const UA = `@oclif/plugin-help/${VERSION} ${process.platform}-${process.arch} node-${process.version}`
+const NAME = require('../package.json').name
+const UA = `${NAME}/${VERSION} ${process.platform}-${process.arch} node-${process.version}`
```

Comment:
Should this commit be pulled out into a separate PR to merge upstream?

Reply:
Yes, there should be 2 separate PRs upstream.